front/cypress/integration/admin/projects/use_content_builder.ts

```diff
@@ -0,0 +1,65 @@
+import { randomString } from '../../../support/commands';
+
+const projectTitleEN = randomString();
+const projectTitleNLBE = randomString();
+const projectTitleNLNL = randomString();
+const projectTitleFRBE = randomString();
+
+describe('Admin: add project and edit description', () => {
+  beforeEach(() => {
+    cy.setAdminLoginCookie();
+  });
+
+  it('creates a draft project', () => {
```

Comment:
I'm not sure a new project is needed every time - maybe you can directly visit the first one in the existing list and skip this step?

Reply:
@IvaKop For sure - that would be much faster. Can I assume there will always be a project in the list?

Reply:
Yes, because it's always using the same template.

Reply:
@IvaKop This ended up being quite flaky, and the conditional check to see whether the toggle was enabled wasn't always accurate. I took a look at some of the other tests and decided to use the cy.apiCreateProject instead, as it's quite fast and seems to be the standard in other e2e tests (so a new project is created for this test, but not via the GUI).

Reply:
You came up with a really nice solution! makes a lot of sense 👍